src/clj/game/cards/operations.clj

```diff
@@ -383,6 +383,25 @@
                                    :effect (req (if tagged
                                                   (damage state side eid :meat 1 {:card card})
                                                   (tag-runner state :runner eid 1)))}}}}}
+   
+   "Eavesdrop"
+    (let [new-sub {:label "[Eavesdrop]: Trace 3 - Give the Runner 1 tag"}]
+      {:implementation "On encounter effect is manual"
+       :sub-effect {:label "Give the Runner 1 tag"
+                   :delayed-completion true
+                   :trace {:base 3
+                           :successful {:msg "give the Runner 1 tag"
+                                        :delayed-completion true
```

Comment:
This key was renamed to `:async`.

Reply:
removed the first `:delayed-completion` and changed the second to `:async`